src/client/Microsoft.Identity.Client/ManagedIdentity/ScopeUtilities.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Text.RegularExpressions;
+
+namespace Microsoft.Identity.Client.ManagedIdentity
+{
+    internal static class ScopeUtilities
```

Comment:
There already exists a ScopeHelper, maybe you can add this functionality to the same class?